docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,116 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
+
+## Security
+
+* Fix potential incorrect rejection when using OAuth 2.0 refresh tokens. ({issue}`13168`)
+* Fix showing outdated OAuth 2.0 refresh tokens in the web UI. ({issue}`13168`)
+
+## Docker image
+
+* Add image for `ppc64le`. ({issue}`13522`)
```

Comment:
We should mention it's not tested so things may not work? WDYT @electrum  ?

Reply:
Could be as simple as saying

>Add experimental image for `ppc64le`.